src/edu/psu/sweng500/team8/gui/CustomKeyListener.java

```diff
@@ -24,7 +24,10 @@ public CustomKeyListener(Cell cell, GameSession gameSession) {
 
 	public void keyTyped(java.awt.event.KeyEvent keyEvent) {
 		
-		if (KeyEvent.VK_BACK_SPACE == keyEvent.getKeyChar()) {
+		
+		
+		if (KeyEvent.VK_BACK_SPACE == keyEvent.getKeyChar() || KeyEvent.VK_DELETE == keyEvent.getKeyChar()) {
+			
 			gameSession.enterNumber(this.cell, 0);
 		} else {
 			gameSession.enterNumber(this.cell, Integer.parseInt(String.valueOf(keyEvent.getKeyChar())));

```

Comment:
I'm going to leave it there until we either unify or David confirms if this collide/not collide with validation logic as it was tested and worked so far.